src/LogViewerHTML.h

```
@@ -0,0 +1,164 @@
+        function fetchLogs() {
+            fetch('/api/logs')
+                .then(function(r) { return r.text(); })
+                .then(function(data) {
+                    logEl.textContent = data || '(empty)';
+                    if (autoEl.checked) logEl.scrollTop = logEl.scrollHeight;
+                })
+                .catch(function(e) { logEl.textContent = 'Error: ' + e.message; });
+        }
+        function copyLogs() {
+            var text = logEl.textContent;
+            if (navigator.clipboard) {
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)